export const unexpectedError = {
  error: 'An unexpected error has occured. Please try again later',
};
